spec/features/static_pages_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+describe "Home", type: :feature  do
+  let!(:card) { Card.create(original_text: "example",
+                            translated_text: "пример")
+              }
```

Comment:
`end` at 5, 14 is not aligned with `let!(:card) { Card.create(original_text: "example",` at 3, 2